packages/service-library/src/data-providers/page-document-provider.ts

```diff
@@ -8,21 +8,23 @@ import { ItemType, RunState, WebsitePage, WebsitePageBase, WebsitePageExtra } fr
 
 @injectable()
 export class PageDocumentProvider {
-    public static readonly pageActiveBeforeDays = 7;
-    public static readonly pageRescanAfterDays = 1;
-    public static readonly rescanAbandonedRunAfterHours = 12;
-    public static readonly maxRetryCount = 3;
+    public static readonly minLastReferenceSeenInDays = 7;
```

Comment:
would be nice to rename to ``minimumLastReferenceSeenInDays`` since ``min`` causes confusion with ``minute``?

Reply:
it makes the variable name to be too long